Sources/Rendering/Core/AbstractMapper3D/index.js

```diff
@@ -16,14 +16,11 @@ function vtkAbstractMapper3D(publicAPI, model) {
     }
   };
 
-  publicAPI.getCenter = (...center) => {
+  publicAPI.getCenter = () => {
     publicAPI.getBounds();
     for (let i = 0; i < 3; i++) {
       model.center[i] = (model.bounds[2 * i + 1] + model.bounds[2 * i]) / 2.0;
     }
-    if (Array.isArray(center[0])) {
-      center[0] = model.center.splice(0);
-    }
     return model.center;
```

Comment:
To be consistent with getArray in macro.js, shouldn't we return a copy of the center ([].concat(model.center)). I'm afraid it could be error prone to return a reference and some developer ends up modifying it.

Reply:
Yes you are right. :P I will make that change.